README.md

```diff
@@ -79,3 +79,22 @@ Run with coverage `python -m pytest --cov=src`
 Run with coverage and missing lines `python -m pytest --cov-report term-missing --cov=src`
 
 Run with coverage, missing lines and junit output `python -m pytest --cov-report term-missing --junitxml="result.xml" --cov=src`
+
+## Yara rules checks.
+
```

Comment:
Is it worth adding some instructions for the Terraform. It's not going to need deploying very much, but it still needs covering in the README I think.

Reply:
I've put a couple of lines in saying what its for.